component/Page/Profile/index.tsx

```diff
@@ -31,6 +31,7 @@ export default function ProfilePage(props: PageProps) {
     const router = useRouter();
     useEffect(() => {
         if (data) {
+            localStorage.setItem(MessageEnum.targetUserName, username);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Storing sensitive data like usernames in local storage can be a security risk. Local storage is accessible by any script on your page, which opens up the possibility of an XSS attack. Consider using HTTP Only cookies or other secure ways to store this information.
```diff
- localStorage.setItem(MessageEnum.targetUserName, username);
+ // Consider using a more secure method to store sensitive data
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->